routes/api/events.js

```diff
@@ -1,9 +1,376 @@
 const express = require('express');
 const router = express.Router();
+const shortid = require('shortid');
+const { check, validationResult } = require('express-validator');
+const auth = require('../../middleware/auth');
+
+const Event = require('../../models/Event');
+const User = require('../../models/User');
 
 // @route   GET api/events
-// @desc    Test route
+// @desc    Get all events
+// @access  Public
+router.get('/', async (req, res) => {

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/spumony/xvent-project-backend/pullRequest?prid=7531905)
```suggestion
router.get("/", async (req, res) => {
```